config/locales/en/errors.yml

```diff
@@ -47,14 +47,6 @@ en:
         heading: Access restricted
         lead_text: This page requires you to be signed in to LAA Portal. You will be redirected back to this service after you sign in.
         sign_in_button: Sign in with LAA Portal
-      not_enrolled:
```

Comment:
I don't think you should take all of this out - there is still the issue of someone giving them the link and they don't have the correct role from portal.

Reply:
Oh I see, I thought that was for something else. Will amend